charts/influxdb-enterprise/values.yaml

```diff
@@ -227,6 +227,7 @@ data:
     # annotations:
     # accessMode: ReadWriteOnce
     # size: 8Gi
+  maxseries: 1500000
```

Comment:
What maxseries do we have in our cloud version? I don't think we have such a limit.

Reply:
1000000